src/shared/components/card/card.tsx

```diff
@@ -0,0 +1,34 @@
+import type { MouseEvent } from 'react';
+
+import * as styles from './card.css';
+
+interface CardProps {
+  title: string;
+  major?: string;
+  description: string;
+  img: string;
+  type: 'details' | 'apply';
+  onClick?: (event: MouseEvent<HTMLDivElement>) => void;
```

Comment:
카드 컴포넌트를 눌렀을 때 그 Card에 대한 상세페이지를 보여주기 위해서는 event 객체는 필요 없지 않을까 싶어요!
```suggestion
  onClick?: () => void;
```
이렇게만 해도 괜찮지 않을까 싶습니다!